README.md

```diff
@@ -23,84 +23,149 @@ Dependencies
 Installation
 -----------
 
-1. Download the python-rtmbot code
+1. Create your project
 
-        git clone https://github.com/slackhq/python-rtmbot.git
-        cd python-rtmbot
+        mkdir myproject
+        cd myproject
 
-2. Install dependencies ([virtualenv](http://virtualenv.readthedocs.org/en/latest/) is recommended.)
+2. Install rtmbot (ideally into a virtualenv https://virtualenv.readthedocs.io/en/latest/)
 
-        pip install -r requirements.txt
+        pip install rtmbot
 
-3. Configure rtmbot (https://api.slack.com/bot-users)
+3. Create and rtmbot.conf file and configure rtmbot (https://api.slack.com/bot-users)
 
-        cp docs/example-config/rtmbot.conf .
-        vi rtmbot.conf
-          SLACK_TOKEN: "xoxb-11111111111-222222222222222"
+        # Add the following to rtmbot.conf
+        DEBUG: True # make this False in production
+        SLACK_TOKEN: "xoxb-11111111111-222222222222222"
+        ACTIVE_PLUGINS:
+            - plugins.repeat.RepeatPlugin
 
-*Note*: At this point rtmbot is ready to run, however no plugins are configured.
+```DEBUG``` will adjust logging verbosity and cause the runner to exit on exceptions, generally making dubugging more pleasant.
+
+```SLACK_TOKEN``` is needed to authenticate with your Slack team. More info at https://api.slack.com/web#authentication
```

Comment:
`[to authenticate with your Slack team](https://api.slack.com/web#authentication).`